tests/suite/regression.rs

```diff
@@ -1889,12 +1889,57 @@ fn regression_test_120() {
 
 #[test]
 fn regression_test_121() {
```

Comment:
This commit seems to remove a test from the `master` branch. Since this commit should not modify any regression test, I think the cleanest solution is to remove `regression.rs` from the commit, and new changes will be merged in their correspondent issues. Thanks!

Reply:
Finally a conflict generated after merging another pull request has fixed the issue, thanks for your work.